hw/ip_templates/racl_ctrl/lint/racl_ctrl.vlt.tpl

```diff
@@ -0,0 +1,5 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// waiver file for ${module_instance_name}
```

Comment:
Are you expecting to need Verilator waivers for the generated modules? If not, I'd be tempted to drop this file.

Reply:
Let's drop that file for now and introduce it if needed later.